terms.html

```diff
@@ -6,6 +6,14 @@
 
 <dl class="termlist">
 
+  <dt><dfn data-lt="amplification">amplification attack</dfn></dt>
+  <dd>
+A class of attack where the attacker attempts to exhaust a target systems
+CPU, storage, or network resources by providing small, valid inputs into the
+system that result in damaging effects that are multiple times more costly
+to process than just the inputs themselves.
```

Comment:
```suggestion
A class of attack where the attacker attempts to exhaust a target systems
CPU, storage, or network resources by providing small, valid inputs into the
system that result in damaging effects that are multiple times more costly
to process than just the inputs themselves.
```
```suggestion
A class of attack where the attacker attempts to exhaust a target system's
CPU, storage, network, or other resources by providing small, valid inputs into
the system that result in damaging effects that can be exponentially more costly
to process than the inputs themselves.
```

It may be worth another sentence with something like `Such effects include triggering infinite processing loops, recursively infinite archive expansion, and transmission of irrational numbers to the last decimal point, among others.`